example/models.go

```diff
@@ -20,8 +25,8 @@ type Product struct {
 type Category struct {
 	Model
 
-	Name    string
-	Type    Type
+	Name string
+	//Type    Type
```

Comment:
we should either remove this or put a warning like "hey, this is not supported yet but it will be"